CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.16)
+
+# Set basic project properties
+project(RCM-f90
+    VERSION 0.1
+    DESCRIPTION "RCM f90 is a modified version of the code available at https://people.math.sc.edu/Burkardt/f_src/rcm/rcm.html to facilitate building as a library, released under an LGPL license - see LICENSE."
+    LANGUAGES Fortran
+)
+
+# Set up install directories
+include(GNUInstallDirs)
+set(INSTALL_INCLUDEDIR "${CMAKE_INSTALL_INCLUDEDIR}/rcm-f90")
+set(INSTALL_MODULEDIR "${INSTALL_INCLUDEDIR}/mod")
```

Comment:
Similarly to the `include/` comments, please install the library under `lib/`.

As discussed yesterday there is some desire to have static/shared library functionality - see https://cmake.org/cmake/help/latest/guide/tutorial/Selecting%20Static%20or%20Shared%20Libraries.html - no strong opinions on the default. The link also details some library path settings.

Reply:
The GNUInstallDirs module sets the library installation path to `lib/`, no need to set explicitly

Reply:
Yup, the GNUInstallDirs should set the install paths for lib and include to their normal paths. I'll make sure that everything is install in the same places as the make version